### PR TITLE
NONPR-1909 Sbt configuration

### PR DESCRIPTION
**Pull request by greg-gajda**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
```
Comment (craigtd) [inline project/plugins.sbt:4]: latest version appears to be 1.7.0. Other plugins are up to date

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      "net.logstash.logback" %  "logstash-logback-encoder"     % "6.1",
+      "fr.davit"             %% "akka-http-metrics-prometheus" % metricsVersion,
+      "io.prometheus"        %  "simpleclient_common"          % prometheusClientsVersion,
+      "io.prometheus"        %  "simpleclient_dropwizard"      % prometheusClientsVersion,
+      "io.prometheus"        %  "simpleclient_hotspot"         % prometheusClientsVersion,
+
+      "com.typesafe.akka"    %% "akka-http-testkit"        % akkaHttpVersion % Test,
+      "com.typesafe.akka"    %% "akka-actor-testkit-typed" % akkaVersion     % Test,
+      "com.typesafe.akka"    %% "akka-stream-testkit"      % akkaVersion     % Test,
+      "org.scalatest"        %% "scalatest"                % "3.2.2"         % Test,
```
Comment (craigtd) [inline build.sbt:57]: I haven't checked all of the above, but scalatest 3.2.7 is available

**Review by craigtd — COMMENTED** [review]
there are some newer versions of dependencies - not sure if you care about this at the moment though

**Review by craigtd — APPROVED** [review]
Approved with feedback

File: project/plugins.sbt (hunk 1 above)
Comment (greg-gajda) [reply project/plugins.sbt:4]: I didn't touch them, but I guess it can be updated

Comment (greg-gajda) [reply project/plugins.sbt:4]: 1.7.0 tag was created 4 hours ago :)

**Pull request merged**